### DISCUSSION TITLE
How to disable escape key behavior on `Editable`

### DISCUSSION THREAD

**Discussion by baptisteArno** [chakra-ui/chakra-ui#5810] [Q&A]
Is there an easy to disable the default behavior of the escape key on an `Editable` component?

Whenever I hit escape, it exists the "edit" mode and keeps the old input value.

I'd like it to submit the input (as the "Enter" key does)

**Comment by chr-ge** [comment]
Unfortunately, no.

**Comment by heycassidy** [comment]
Have you tried the submitOnBlur prop?
